js/sign/README.md

```diff
@@ -104,14 +104,18 @@ This package also includes 2 CLI tools
 There are the following command-line flags available:
 
 - (required) `--privateKey <filePath>` (`-k <filePath>`)  
```

Comment:
Could you please fix this too? It's now `--private-key` in kebab-case.

Reply:
Fixed in the previous one as well.